frontend/views/pages/Join.vue

```diff
@@ -149,7 +137,13 @@ export default ({
           creatorPicture,
           message
         }
-        this.pageStatus = 'SIGNING'
```

Comment:
No need to set `this.pageStatus = 'SIGNING'`. We should only set this when the user is not logged in.
So replace this to the next line below inside the `if` statement.

Reply:
With the code like this https://github.com/okTurtles/group-income/pull/1603#discussion_r1206179653

Then it makes sense to have this.